src/browser/cacheDetail.js

```diff
@@ -26,7 +26,15 @@ function CacheDetailController($scope, SimDriver) {
         ctrl.cacheInfo = data;
     });
 
-    ctrl.getAssociativity = function(event, index) {
+    ctrl.getAssociativity = function(event) {
         return parseInt(ctrl.cacheInfo.caches[ctrl.index].associativity);
     }
+
+    ctrl.getIndex = function(event) {
```

Comment:
this works but I wanna look into how we can use the functions already defined else where

Reply:
oh nevermind this is actually how many different indices there are. if thats the case can you rename the function since the other `getIndex` serves a different purpose